### ISSUE TITLE
did MQTT change in latest update?

### ISSUE THREAD

**Issue by BertoTavira** [SeydX/homebridge-camera-ui#101]
Hey @SeydX, thanks again for the plugin, here again trying to solve something (or maybe help others to solve) :)

**Describe the bug**
Motion sensors are activated but instantly disappear in Home app, they don't stay activated while there's movement, like they did before update.

**To Reproduce**
- updated plugin and kept the parameters: didn't work, motion sensor disappear after 1 sec (default parameter)
- tried with "motionTimeout" to 0, 1 or any number of seconds: motion sensor appears, stays for the seconds previously added, then disappear.
- tried to change the MQTT config parameters as said in [Wiki](https://sunoo.github.io/homebridge-camera-ffmpeg/automation/mqtt.html): same result (BTW they're different than Camera-UI plugin)

My guess:
- Seems camera cannot reset sensor, maybe related to [this ](https://github.com/Sunoo/homebridge-camera-ffmpeg/issues/877) and/or [this](https://github.com/Sunoo/homebridge-camera-ffmpeg/issues/1023)

- Seems also if "motionTimeout" is set to any number, plugin resets the sensor but if camera still detecting movement, it won't send a new ON message until it send its own OFF

**Expected behavior**
Motion sensor to be activated, stay on while there's movement, and be off when there's none

**Logs**
I can see ON and OFF messages in log
Also checked with a MQTTAnalizer for iOS and all messages are there
so it looks like plugin or homebridge cannot handle them well and reset the motion sensor

**Environment**
- Node.js Version: v14.16.0
- Homebridge Version: v1.3.4
- Camera UI Version: v3.2.3
- Operating System: rpi homebridge image

**Comment by SeydX** [comment]
@BertoTavira 

can u post your config.json pls

**Comment by BertoTavira** [comment]
Sure:

```
{
            "name": "CameraUI",
            "port": 8181,
            "auth": "form",
            "theme": "auto",
            "language": "auto",
            "debug": true,
            "reset": false,
            "mqtt": {
                "active": true,
                "host": "192.168.31.238",
                "port": 1883,
                "motionTopic": "homebridge",
                "motionMessage": "ON",
                "motionResetTopic": "homebridge",
                "motionResetMessage": "OFF"
            },
            "http": {
                "active": false,
                "localhttp": false
            },
            "ssl": {
                "active": false
            },
            "options": {
                "videoProcessor": "ffmpeg"
            },
            "cameras": [
                {
                    "name": "YiHome",
                    "manufacturer": "Yi",
                    "model": "Home 1080p",
                    "motion": true,
                    "motionTimeout": 0,
                    "unbridge": false,
                    "videoConfig": {
                        "source": "-re -i rtsp://192.168.31.239/ch0_0.h264",
                        "stillImageSource": "-i http://192.168.31.239:8080/cgi-bin/snapshot.sh?res=high&watermark=yes",
                        "maxStreams": 10,
                        "vcodec": "copy",
                        "audio": false,
                        "socketPort": 8100
                    }
                },
                {
                    "name": "Dome",
                    "manufacturer": "Yi",
                    "model": "Cloud Dome 1080p",
                    "motion": true,
                    "motionTimeout": 0,
                    "unbridge": false,
                    "videoConfig": {
                        "source": "-re -i rtsp://192.168.31.106/ch0_0.h264",
                        "stillImageSource": "-rtsp_transport tcp -re -i rtsp://192.168.31.106/ch0_0.h264",
                        "maxStreams": 10,
                        "vcodec": "copy",
                        "audio": false,
                        "socketPort": 8200
                    }
                }
            ],
```

**Comment by BertoTavira** [comment]
If it helps, I tried with topic "homebridge/motion" or just "hombridge"
also tried your plugin parameters vs camera/ffmpeg for MQTT, combined with "motionTimeout",
but couldn't find the way to make it work as before :(

**Comment by SeydX** [comment]
@BertoTavira 

Atm, mqtt doesnt changed. Pls dont follow the instructions on wiki. It should be the same config as before.

There will be some major changes on next version

**Comment by BertoTavira** [comment]
Thanks for checking, @SeydX

I’ll revert to old config and wait for the update and see if this goes back to work again

**Comment by SeydX** [comment]
Fixed in [v3.3.0 Beta](https://github.com/SeydX/homebridge-camera-ui/releases/tag/3.3.0)
